Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and ('$(BuildingInsideVisualStudio)' != 'true' or '$(RUNTIME_PATH)' != '')">%RUNTIME_PATH%\</TestRuntimeEnvVar>
     <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(TestRuntimeEnvVar)' == ''">$(TestSharedFxDir)\</TestRuntimeEnvVar>
```

Comment:
Can we limit it to just the StartProgram which is an easy concat `$(TestSharedFxDir)\dotnet.exe` and for the test assembly can we eliminate giving it the ExecutionPath all together and instead depend on the working directory to find it?